src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
+  display: flex;
+  justify-content: center;
   margin: 0;
+  background: #fff;
+  font-family: "Roboto", sans-serif;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
All `reset type` styles better to move to the top of the file to separate from the nest